rls/src/main/java/io/grpc/rls/RlsLoadBalancerProvider.java

```diff
@@ -62,12 +62,15 @@ public ConfigOrError parseLoadBalancingPolicyConfig(Map<String, ?> rawLoadBalanc
     try {
       RouteLookupConfig routeLookupConfig = new RouteLookupConfigConverter()
           .convert(JsonUtil.getObject(rawLoadBalancingConfigPolicy, "routeLookupConfig"));
+      Map<String, ?> routeLookupChannelServiceConfig =
+          JsonUtil.getObject(rawLoadBalancingConfigPolicy, "routeLookupChannelServiceConfig");
```

Comment:
If we allow `rawLoadBalancingConfigPolicy` to be null (see my comment above), this will throw.

Reply:
I thought `rawLoadBalancingConfigPolicy` should never be null. Did you see it is null anywhere?

Reply:
Ah, sorry, I meant `routeLookupChannelServiceConfig`, copy-paste from a wrong place.

Reply:
`routeLookupChannelServiceConfig` can be null. Did you mean this line will throw or code below will throw?

Reply:
JsonUtil.getObject says "If the value is not a Map, throws an exception." So in our case it could be `null`, therefore an exception.

Reply:
It say "If the key is not present, this returns null. If the value is not a Map, throws an exception." What it really means  is "If the value is not null and the type is not Map, throws an exception". The javadoc may not be accurate but the implementation can confirm this.

Reply:
I added a test to verify the behavior. Looks like it throws when the map contains a null value for a given key: https://github.com/grpc/grpc-java/pull/8881.
Or am I misreading the issue here? I don't know how `rawLoadBalancingConfigPolicy` map looks like. Should we maybe add a unit test for `parseLoadBalancingPolicyConfig()` to demonstrate the inputs?

Reply:
Ah, now I understand what you mean: the map contains the key but the value is null. I think that might be a bug for JsonUtil. cc @ejona86.  Currently we fail to parse lb configs if there is an explicit `null` value in the config. `JsonUtil.getObject()` or other methods are widely used this way and will have the same issue.

Reply:
OK. Let me know if we need to follow up in a separate PR.

Reply:
Filed #8883 to track this issue. Thanks @sergiitk.

Reply:
Why do you say "routeLookupChannelServiceConfig can be null"? It can be "not present", but why could it be `null`?